ci-operator/config/openshift/knative-serving/openshift-knative-serving-release-v0.8.1.yaml

```
@@ -0,0 +1,199 @@
+tag_specification:
+  name: '4.1'
+  namespace: ocp
+promotion:
+  cluster: https://api.ci.openshift.org
+  namespace: openshift
+  name: release-v0.8.1
+base_images:
+  base:
+    name: '4.1'
+    namespace: ocp
+    tag: base
+build_root:
+  project_image:
+    dockerfile_path: openshift/ci-operator/build-image/Dockerfile
+canonical_go_repository: knative.dev/serving
+binary_build_commands: make install
+test_binary_build_commands: make test-install
+tests:
+- as: e2e-aws
```

```suggestion
- as: e2e-aws-ocp-41
```

As in release-next please.